test/lib/view.js

```diff
@@ -332,6 +332,13 @@ t.test('should log package info', t => {
     })
   })
 
+  t.test('package with semver range', t => {
+    view.exec(['blue@^1.0.0'], () => {
+      t.matchSnapshot(logs)
```

Comment:
We should set blue's time object up so that this snapshot tests what we want here.

So before mocks we can set up 
```javascript
const yesterday = new Date(new Date() - 1000 * 60 * 60 * 25)
```

and blue's time object can be

```javascript
      time: {
        '1.0.0': yesterday,
      },
```

And then our snapshots for blue will all say `published yesterday`

Reply:
Adjusted.